### PR TITLE
Manju new

### PR DESCRIPTION
**Pull request by sudoManju**
Requesting for pull to merge

### CODE REVIEW

File: pages/citizen/registration.py [hunk 1]
```diff
@@ -31,13 +35,27 @@ def profile(self):
...
+
 
 @PageObject
 class OTPPage(Page):
     class ID:
         txtOTP = "input#otp"
         btnGetStarted = "button#otp-start"
         btnResend = "div#otp-resend"
+        lblOTP = "xpath=//label[@for='otp']/following-sibling::div[text()[1]]"
+        lblOTPSentTo = "xpath=//div[@class='label-text otp-mobile-number']"
```
Comment (tarunlalwani) [inline pages/citizen/registration.py]: Use CSS here

File: pages/citizen/registration.py [hunk 2]
```diff
@@ -31,13 +35,27 @@ def profile(self):
...
+        return get(self.ID.lblUserNameValidation)
+
 
 @PageObject
 class OTPPage(Page):
     class ID:
         txtOTP = "input#otp"
         btnGetStarted = "button#otp-start"
         btnResend = "div#otp-resend"
+        lblOTP = "xpath=//label[@for='otp']/following-sibling::div[text()[1]]"
```
Comment (tarunlalwani) [inline pages/citizen/registration.py]: Why is OTP label being used? If being read later then only use it

File: environment/__init__.py [hunk 3]
```diff
@@ -16,3 +16,4 @@ def to_absolute_path(path):
 DEFAULT_IMAGELIST_ONE = DEFAULT_IMAGELIST[:1]
 DEFAULT_IMAGELIST_TWO = DEFAULT_IMAGELIST[:2]
 DEFAULT_IMAGELIST_THREE = DEFAULT_IMAGELIST[:3]
+PROFILE_IMAGE = DEFAULT_IMAGELIST[3:]
```
Comment (tarunlalwani) [inline environment/__init__.py]: Use a path just for profile image. Make sure to use relative path only and resolve it later in init

File: pages/citizen/registration.py [hunk 4]
```diff
@@ -31,13 +35,27 @@ def profile(self):
         click(self.ID.btnProfile)
         return self
 
+    def error_message(self):
```
Comment (tarunlalwani) [inline pages/citizen/registration.py]: Should be `get_mobileno_error_message`

File: pages/citizen/registration.py [hunk 5]
```diff
@@ -14,9 +14,13 @@ class ID:
         txtMobileNumber = "input#person-phone"
         btnLogin = "button#login-submit-action"
         btnProfile = "#header-profile"
+        lblMobileNumber = "//label[contains(text(), 'Mobile Number')]"
+        lblRequired = "xpath=//label[@for='person-phone']/following-sibling::div[last()]"
```
Comment (tarunlalwani) [inline pages/citizen/registration.py]: This should be `lblMobileNumberErrrorMessage`

File: pages/citizen/registration.py [hunk 6]
```diff
@@ -51,6 +69,21 @@ def resend(self):
...
+        return get(self.ID.lblOTPSentTo)
+
+    def otp_required(self):
+        return get(self.ID.lblErrorMsg)
+
+    def invalid_otp(self):
+        return get(self.ID.lblErrorMsg)
+
+    def otp_has_resent(self):
+        return get(self.ID.lblResentOTPMsg)
```
Comment (tarunlalwani) [inline pages/citizen/registration.py:82]: Should use wait for toast and return that message

File: pages/flows/common.py [hunk 7]
```diff
@@ -44,105 +82,96 @@ def create_new_complaint_by_plus_icon(location, additional_details, complaint_ty
...
         complain.append(i.get_complaint_no())
-    if complaint_number == 0:
-        for cn in complain:
-            print(cn)
-    else:
-        a = complain.index(complaint_number[0])
-        cards[a].track_complaint()
+
+    complaints = complain.index(complaint_number)
+    cards[complaints].track_complaint()
```
Comment (tarunlalwani) [inline pages/flows/common.py]: Use open_complaint_number

File: tests/test.py [hunk 8]
```diff
@@ -1,44 +1,65 @@
...
-from pages.flows.common import create_new_complaint
 
 
 @fixture(autouse=True, scope='session')
 def my_fixture():
     # setup_stuff
     yield
     try:
-        quit_driver()
+        # quit_driver()
```
Comment (tarunlalwani) [inline tests/test.py]: Don't commit such changes

**Review by tarunlalwani — CHANGES_REQUESTED** [review]
Please work on the feedback

File: environment/__init__.py (hunk 3 above)
Comment (sudoManju) [reply environment/__init__.py]: Done

File: pages/citizen/registration.py (hunk 4 above)
Comment (sudoManju) [reply pages/citizen/registration.py]: Done

File: pages/citizen/registration.py (hunk 6 above)
Comment (sudoManju) [reply pages/citizen/registration.py:82]: Done

File: pages/flows/common.py (hunk 7 above)
Comment (sudoManju) [reply pages/flows/common.py]: Done

File: tests/test.py (hunk 8 above)
Comment (sudoManju) [reply tests/test.py]: Done

File: pages/citizen/registration.py (hunk 1 above)
Comment (sudoManju) [reply pages/citizen/registration.py]: Not using this element on test case anymore.

File: pages/citizen/registration.py (hunk 2 above)
Comment (sudoManju) [reply pages/citizen/registration.py]: OTP label is no longer is using in the test case. Initially it was used for assertion.

File: pages/citizen/registration.py (hunk 5 above)
Comment (sudoManju) [reply pages/citizen/registration.py]: Done

**Comment by sudoManju** [comment]
As per feedback, changes committed on manju_new branch.

Profile photo upload, and some minor changes updated.

Please verify and approve the request.

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**